tests/acceptance/features/bootstrap/TestingAppContext.php

```diff
@@ -185,6 +185,23 @@ public function theAdministratorDeletesTheConfigKeyInAppUsingTheTestingApi($key,
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @Then the last login date for the user should be :day days ago
+	 *
+	 * @param int $day
+	 *
+	 * @return void
+	 */
+	public function theLastLoginDateForUserShouldBe($day) {
+		$responseXml = HttpRequestHelper::getResponseXml(
+			$this->featureContext->getResponse()
+		);
+		$lastLoginTimeStamp = (string)$responseXml->data[0]->element;
+		$currentTimeStamp = \time();
+		$daysAgo = ($currentTimeStamp - $lastLoginTimeStamp) / (24 * 60 * 60);
+		Assert::assertEquals($day, \floor($daysAgo), 'Expected ' . $day . ' days but got ' . $daysAgo . ' days');
```

Comment:
Note: this should work OK, `$daysAgo` should be exactly or a few seconds more than the requested number of days (e.g. 5.0001 days) and `floor` will get us the `5`. It only breaks if the test hangs for a whole 24 hours between set and get steps.